client/src/main/ipc/server.ts

```diff
@@ -7,6 +7,9 @@ import PluginManager from "./plugin-manager";
 import RendererBridge from "../bridge";
 import Stream from "../stream/stream";
 import { core } from "../../gen/core";
+import Log from "../log";
+
+const iconMaxLen = 20000;
```

Comment:
We try to avoid acronyms and abbreviations, how about `iconMaximumLength` or `maximumIconSize`?